spec/acceptance/reporting_spec.rb

```diff
@@ -55,54 +55,83 @@
     master.idempotent_apply(setup_manifest)
   end
 
```

Comment:
Hmm, now that we've defined `incident_creation_report_attributes` and the incident creation condition to be "a report with at least _one_ of the incident creation attributes", some of these tests should be revisited because they were written under our (faulty) initial assumption of `incident_creation_report_statuses`. I'll outline which tests in the other comments.

Reply:
The general thing I'm outlining is to reword the tests to be more in line with how we want users to think about our new `report_attributes` param and the incident creation condition. So it's mostly what you've already got, just slightly moving stuff around.

Reply:
Resolving b/c not a PR blocker